front/src/applications/stdcm/views/OSRDCStdcmConfig.tsx

```diff
@@ -36,18 +37,37 @@ export default function OSRDConfig({
   const scenarioID = useSelector(getScenarioID);
   const timetableID = useSelector(getTimetableID);
   const infraID = useSelector(getInfraID);
+  const selectedTrain = useSelector(getSelectedTrain);
+  const simulation = useSelector(getPresentSimulation);
   const [showMap, setShowMap] = useState<boolean>(true);
+  const [isInfraLoaded, setIsInfraLoaded] = useState<boolean>(false);
 
   const { t } = useTranslation([
     'translation',
     'operationalStudies/manageTrainSchedule',
     'simulation',
   ]);
 
-  const selectedTrain = useSelector(getSelectedTrain);
-  const simulation = useSelector(getPresentSimulation);
+  const { data: infra } = osrdEditoastApi.useGetInfraByIdQuery(
+    { id: infraID as number },
+    {
+      skip: !infraID,
+      pollingInterval: !isInfraLoaded ? 1000 : undefined,
+    }
+  );
+
   const shouldDisplayStdcmResult = simulation.trains[selectedTrain] !== undefined;
 
+  useEffect(() => {
+    if (infra && infra.state === 'NOT_LOADED') {
+      setIsInfraLoaded(false);
+    }
+
+    if (infra && infra.state === 'CACHED') {
+      setIsInfraLoaded(true);
+    }
```

Comment:
you should use an enum to list `infra.state` states common for all test about this

Reply:
There is already an enum for the state property sent by the back end (can be seen in openapi).
I can add one in the front if you want but maybe for a fix later in another issue ?